recipes/lofreq/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eu -o pipefail
+
+./configure --with-htslib=${PREFIX}/lib --prefix=${PREFIX}
```

Comment:
That's not doing anything for you; it should be just `--with-htslib=${PREFIX}`. But I would suggest following the samtools and bcftools recipes and using `--with-htslib=system`.

Reply:
Thank you, I'm still learning the best practice with these libraries, and I struggle with the myriad settings for compiling complex C/C++ packages like these.

Reply:
Sorry, I should have been more explicit: keep the `--prefix` too.

```
./configure --with-htslib=system --prefix=$PREFIX
```